apps/publikator/components/editor/modules/document/createPasteHtml.js

```diff
@@ -28,45 +28,54 @@ const PARAGRAPH_TYPES = ['PARAGRAPH', 'INFOP', 'BLOCKQUOTEPARAGRAPH']
   TEXT
 } = TRANSFER_TYPES*/
 
-export default (centerModule, figureModule) => (event, change, editor) => {
-  const transfer = getEventTransfer(event)
-  const cursor = editor.value.selection.anchorKey
-  const blockType = editor.value.document.getClosestBlock(cursor).type
+const createPasteHtml =
```

Comment:
FYI, most of the lines below are formatting changes because ESLint didn't like that we're exporting an anonymous function.